scripts/main.py

```diff
@@ -36,22 +35,25 @@ def dates_to_process(seed_dates_path):
 
     return date_list
 
+"""
+
 
 # 3.
 # Get JSON files for dates
 
+#upload files to gdrive
+
+service = build("drive", "v3")
```

Comment:
````suggestion
service = googleapiclient.discovery.build("drive", "v3")
````

see change in line 11. in general, i prefer to have the full name of the library which makes debugging slightly easier when there's something wrong (that i can search the documentation more easily.